templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -410,6 +416,22 @@ Parameters:
     ConstraintDescription: Must be 'General Purpose (SSD)' or 'Provisioned IOPS'.
     Description: Database storage type.
     Type: String
+  DBAllowMajorVersionUpgrade:
```

Comment:
I feel we need more documentation on this as this is a new feature and description on fields may not be enough to explain the logic and workflow to the users.